README.md

```diff
@@ -1,154 +1,223 @@
+[![Twitter](https://img.shields.io/badge/Twitter-%231DA1F2.svg?style=for-the-badge&logo=Twitter&logoColor=white)](https://twitter.com/PraisesPJMT/)
+
+👤 **Nonhlanhla**
+
+- GitHub: [@githubhandle](https://github.com/29td)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/nonhlanhla-mndebele-ab7448226/)
+
+
 👤 **Youta Christabelle**
 
+- GitHub: [Christabelle143](https://github.com/Christabelle143)
+- Twitter: [@christabelleyo](https://twitter.com/christabelleyo)
+- LinkedIn: [YoutaChristabelle](https://linkedin.com/in/YoutaChristabelle)
+
+👤 **Okolike Assumpta**
 
+- GitHub: [@githubhandle](https://github.com/summyalena)
+- Twitter: [@twitterhandle](https://twitter.com/AOkolike)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/okolike-assumpta)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## 🔭 Future Features <a name="future-features"></a>
 
-> The following are features to be expected in the future
+## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **Connect to more API endpoints**
+- Rotation of car models
+- Improvements on UI design
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
 ## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/edi-sipka/Vip_Cars_Rails/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
+
 ## ⭐️ Show your support <a name="support"></a>
 
-Give a ⭐️ if you like this project!
+If you like this project buy the team a [drink](.../.../)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
+
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> - Original UI design by [Gregorire Vella](https://www.behance.net/gregoirevella) on [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=).
-> - Tip hart to you for checking this project out
+I would like to acknowledge:
+
+- [Microverse](https://www.microverse.org/) for the opportunity to work on this project
+- [The creative common](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) for the design
+- Design from [Murat Korkmaz on Behance](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign)
+
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
+
+## ❓ FAQ <a name="faq"></a>
+
+- **How I can run this project?**
+
+  - After cloning repository, run `npm install` to install all dependencies. Then run `npm start` to run the project.
+
+- **How I can run tests?**
+
+  - After cloning repository, run `npm install` to install all dependencies. Then run `npm test` to run the tests.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪

Reply:
Well noted! The Licence link is now fixed!